storage/remote/queue_manager.go

```diff
@@ -376,17 +428,17 @@ func (t *QueueManager) reshard(n int) {
 
 type shards struct {
 	qm      *QueueManager
-	queues  []chan *model.Sample
+	queues  []chan tsdb.RefSample
 	done    chan struct{}
 	running int32
 	ctx     context.Context
 	cancel  context.CancelFunc
 }
 
 func (t *QueueManager) newShards(numShards int) *shards {
-	queues := make([]chan *model.Sample, numShards)
+	queues := make([]chan tsdb.RefSample, numShards)
 	for i := 0; i < numShards; i++ {
-		queues[i] = make(chan *model.Sample, t.cfg.Capacity)
+		queues[i] = make(chan tsdb.RefSample, 10)
```

Comment:
Lets either make this a const at the top of file, of put it back to t.cfg.Capacity and set that to 10 by default.

Reply:
exporting `Capacity` on the `QueueConfig` struct makes it configurable by the user correct? we don't want it to be configurable anymore